packages/components/ng-ovh-payment-method/src/components/register/index.html

```diff
@@ -216,3 +159,131 @@
+</script>
+<script type="text/ng-template" id="sepaExplanationTextIntro.html">
+    <div class="text-justify">
+        <p
+            class="mb-0"
+            data-translate="ovh_payment_method_register_explanation_sepa_direct_debit_warning_redirection"
+        ></p>
+        <p
+            class="mb-0"
+            data-translate="ovh_payment_method_register_explanation_sepa_direct_debit_warning_order"
+        ></p>
+        <p
+            class="mb-0"
+            data-translate="ovh_payment_method_register_explanation_sepa_direct_debit_warning_owner_identity"
+        ></p>
+    </div>
+</script>
+<script type="text/ng-template" id="sepaExplanationTextBanner.html">
+    <oui-message
+        data-type="{{ $ctrl.defaultExplanationTextsOptions.sepa.banner || 'info' }}"
+        class="mt-2 text-justify"
+    >
+        <p
+            class="mb-0"
+            data-translate="ovh_payment_method_register_explanation_sepa_direct_debit_data_processing_info"
+        ></p>
+        <p
+            class="mb-0"
+            data-translate="ovh_payment_method_register_explanation_sepa_direct_debit_digital_signature_info"
+        ></p>
+    </oui-message>
+</script>
+
+<!-- paypalExplanationText -->
+<script type="text/ng-template" id="paypalExplanationTextContainer.html">
+    <ng-include
+        data-ng-if="!$ctrl.defaultExplanationTextsOptions.paypal"
+        data-src="'paypalExplanationText.html'"
+    />
+    <oui-message
+        data-ng-if="$ctrl.defaultExplanationTextsOptions.paypal"
+        data-type="{{ $ctrl.defaultExplanationTextsOptions.paypal }}"
+    >
+        <ng-include data-src="'paypalExplanationText.html'" />
+    </oui-message>
+</script>
+<script type="text/ng-template" id="paypalExplanationText.html">
+    <span
+        data-ng-if="$ctrl.coreConfig.isRegion('US')"
+        data-translate="ovh_payment_method_register_explanation_selected_paypal_redirect_US"
+    ></span>
+    <span
+        data-ng-if="!$ctrl.coreConfig.isRegion('US')"
+        data-translate="ovh_payment_method_register_explanation_selected_paypal_redirect"
+    ></span>
+</script>
+
+<!-- rupayExplanationText -->
+<script type="text/ng-template" id="rupayExplanationTextContainer.html">
+    <ng-include
+        data-ng-if="!$ctrl.defaultExplanationTextsOptions.rupay"
+        data-src="'creditCardChargesExplanationText.html'"
+    />
+    <oui-message
+        data-ng-if="$ctrl.defaultExplanationTextsOptions.rupay"
+        data-type="{{ $ctrl.defaultExplanationTextsOptions.rupay }}"
+    >
+        <ng-include data-src="'creditCardChargesExplanationText.html'" />
+    </oui-message>
+</script>
+<script type="text/ng-template" id="creditCardChargesExplanationText.html">
+    <p
+        data-translate="ovh_payment_method_register_explanation_credit_card_registration_charges_info"
+        data-translate-values="{
+            registrationCharges: $ctrl.registrationCharges
+        }"
+    ></p>
+</script>
```

Comment:
**nitpick:** Instead of creating an `ng-template` can you create a folder called `templates` and add html file for each type so that we can have a better readability.